python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -145,14 +147,18 @@ def _executeJob(
 
         Parameters
         ----------
-        quantumExecutor : `QuantumExecutor`
-            Executor for single quantum.
-        task_node : `lsst.pipe.base.pipeline_graph.TaskNode`
-            Task definition structure.
+        quantumExecutor_pickle : `bytes`
+            Executor for single quantum, pickled.
+        task_node : `bytes`
```

Comment:
```suggestion
        task_node_pickle : `bytes`
```
(I think numpydoc ignored the problem because it's a private method)